dtcli/utilities/cadcclient.py

```diff
@@ -98,13 +112,16 @@ def get(
             logger.debug(f"{filename} ➜ {destination[index]} ✔")
     except cadcutils.exceptions.NotFoundException as error:  # type: ignore
         logger.error(f"CADC Exception: {error}")
-        raise error
+        not_found.append(str(error))
```

Comment:
why are you appending the error instead of the file names? We are already logging the error here.